### PR TITLE
Add Pate Chinois recipe

### PR DESCRIPTION
**Pull request by jlevesque**

### CODE REVIEW

File: src/pate-chinois.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Pâté Chinois (Quebec Style Shepherd’s Pie)
+
+This is the most traditional dish that every household eats in Quebec. As a kid, you would eat Pâté Chinois at least once a week, either at home or at the school's cafeteria,. It is very similar to a Shepherd’s Pie, as it is divided in 3 layers of ingredients: meat, legumes and mashed potatoes. The literal translation of the name is something like _chinese pie_. You can search for the origin of the name on the internet, I'm not an history teacher.
+
+
+- ⏲️ Prep time: 20 min
```
Comment (hallzy) [inline src/pate-chinois.md:6]:
```suggestion
This is the most traditional dish that every household eats in Quebec. As a kid, you would eat Pâté Chinois at least once a week, either at home or at the school's cafeteria,. It is very similar to a Shepherd’s Pie, as it is divided in 3 layers of ingredients: meat, legumes and mashed potatoes. The literal translation of the name is something like _chinese pie_. You can search for the origin of the name on the internet, I'm not an history teacher.

- ⏲️ Prep time: 20 min
```

Comment (hallzy) [reply src/pate-chinois.md:6]: Please remove extra empty line

File: src/pate-chinois.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* 1 tbsp of butter
+* 1 lb of ground beef
+* 1 onion, diced
+* 1 tbsp of soya sauce
+* 1 tbsp of herbs de Provence
+
+* 1 tsp of Worcestershire sauce
+
+
+### Corn layer
```
Comment (hallzy) [inline src/pate-chinois.md:32]:
```suggestion
* 1 tbsp of herbs de Provence
* 1 tsp of Worcestershire sauce

### Corn layer
```

Please remove extra empty line, and the empty line between 2 ingredients

File: src/pate-chinois.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Instructions
+
+1. Preheat the oven to 375°F (190°C).
+2. In a large pot of salted water, cook the potatoes for 20-30min or until tender. Drain.
+3. Add the butter, the milk, the herbs, the nutmeg and mash the potatoes with a masher or an electric mixer. Gradually
+add the cheese. Season with salt and pepper and set aside.
+4. In a pan at medium-high heat, brown the onion in the butter and cook the beef until golden brown. Add the herbs, the Worcestershire and the soya sauce and cook it for another 2min. Set aside.
+5. In a 6-8 inch baking dish, press the meat into the bottom. Evenly cover with the whole kernel corn, the cream corn and finish with the layer of mashed potatoes.
+6. Sprinkle the some paprika to your taste.
```
Comment (hallzy) [inline src/pate-chinois.md:44]:
```suggestion
6. Sprinkle some paprika to taste.
```

File: src/pate-chinois.md (hunk 1 above)
Comment (jlevesque) [reply src/pate-chinois.md]: Extra line removed

File: src/pate-chinois.md (hunk 2 above)
Comment (jlevesque) [reply src/pate-chinois.md]: Extra line removed

File: src/pate-chinois.md (hunk 3 above)
Comment (jlevesque) [reply src/pate-chinois.md]: Lazy typo removed

File: src/pate-chinois.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Pâté Chinois (Quebec Style Shepherd’s Pie)
+
```
Comment (Knarkzel) [inline src/pate-chinois.md:2]: Missing image.

File: src/pate-chinois.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+* 1 onion, diced
+* 1 tbsp of soya sauce
+* 1 tbsp of herbs de Provence
+* 1 tsp of Worcestershire sauce
+
+### Corn layer
+* 1 small can (around 14oz) of whole kernel corn
+* 1 small can (around 14oz) of cream style corn
+
+## Instructions
```
Comment (Knarkzel) [inline src/pate-chinois.md:32]: Rename `Instructions` to `Directions`.

File: src/pate-chinois.md (hunk 4 above)
Comment (jlevesque) [reply src/pate-chinois.md:2]: I personally don't have any pictures of the meal yet, any light image for Google would do?

Comment (hallzy) [reply src/pate-chinois.md:2]: Luke doesn't want google images on this site... only images that have been taken by someone that has actually made the dish.

**Comment by notarock** [comment]
Basé ❤️

File: src/pate-chinois.md (hunk 4 above)
Comment (jlevesque) [reply src/pate-chinois.md:2]: Found one I took a while ago, it's not the prettiest thing to look at haha

File: src/pate-chinois.md (hunk 5 above)
Comment (jlevesque) [reply src/pate-chinois.md]: Renamed

**Review by hallzy — APPROVED** [review]

**Review by Knarkzel — APPROVED** [review]

**Pull request merged**